sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -63,6 +63,37 @@ class HiveQuerySuite extends HiveComparisonTest with BeforeAndAfter {
     }
   }
   
+  test("SPARK-5324 Results of describe can't be queried") {
```

Comment:
nit: I'd prefer that the test name said what the *correct* behavior is, rather than the old bug it is checking for.  I think its better when we're looking at test output, or if there is a regression on this and a dev needs to figure out what is being tests.

Maybe I am wrong about this, though -- we haven't applied that universally so far